js/book.js

```diff
@@ -0,0 +1,9 @@
+class Book {
+  constructor(title = 0, author = 0, id = null) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+}
```

Comment:
[ Optional Changes ]

- It would be nice if you could make the title and author be empty strings as they actually meant to be strings. A title of a book is not supposed to be a number nor the name of the author of the book.
- Also, it would be nice if you clear the fields after a book is added so one can add another book without clearing the fields manually. That helps for the good user experience of your app👍

**See the output when creating a book**
![image](https://user-images.githubusercontent.com/50941074/184183377-9aec9963-5692-4e82-bc03-3e38465d4169.png)